ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -783,13 +932,14 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)";
+				PRODUCT_BUNDLE_IDENTIFIER = "NFMTHAZVS9.com.drewvolz.stolaf-";
```

Comment:
This does need to change to match what Apple has on file for the app since we are publishing as an update rather than a new app.